testutil/keeper/application.go

```diff
@@ -18,8 +20,15 @@ import (
 	mocks "pocket/testutil/application/mocks"
 	"pocket/x/application/keeper"
 	"pocket/x/application/types"
+	gatewaytypes "pocket/x/gateway/types"
 )
 
+var AddrToPubKeyMap map[string]cryptotypes.PubKey
+
+func init() {
+	AddrToPubKeyMap = make(map[string]cryptotypes.PubKey)
+}
```

Comment:
Can this not be simplified to?:

```suggestion
var AddrToPubKeyMap = make(map[string]cryptotypes.PubKey)
```